app/models/manageiq/providers/openstack/storage_manager/cinder_manager/cloud_volume.rb

```diff
@@ -227,6 +227,13 @@ def with_provider_connection
     super(cinder_connection_options)
   end
 
+  def self.cinder_connection_options(cloud_tenant = nil)
+    connection_options = {:service => "Volume"}
+    connection_options[:tenant_name] = cloud_tenant.name if cloud_tenant
+    connection_options[:proxy] = openstack_proxy if openstack_proxy
+    connection_options
+  end
+
```

Comment:
Why make this public rather than adding `private_class_method` ?  It looks like the intent was for this to be private given that it is references from a private instance method.

Reply:
Unfortunately if it's made a private_class_method, then line 253 won't work.  As far as I'm aware there's no way to make a `protected_class_method`, which is effectively what we want.  That being said, I don't like moving this up in the file as the solution, as it was clearer when in locality to the caller.

Reply:
I wonder if it's almost better to just make it a private instance method.

Reply:
I've updated it to use `send` internally so that 253 continues to work. I tried to just get rid of the singleton method, but that caused spec failures, and I didn't want risk breakage elsewhere.

Reply:
I don't think making something private then changing its caller to use `.send()` is right and it can't be just a private instance method because `.raw_create_volume` also needs it as a class method.

@Fryguy it seems like duplicating this method at the instance and class level (:-1:) or leaving it as a public class method seem like the only options

Reply:
Or just remove it from this PR, focusing on syntactical changes, and doing the semantic changes in a separate PR.